client/config/index.js

```diff
@@ -0,0 +1,11 @@
+let config;
+
+if ( typeof window !== 'undefined' && window.config ) {
+	config = window.config;
+	config.anyEnabled = window.anyEnabled;
+	config.isEnabled = window.isEnabled;
+} else {
+	throw new Error( 'config not available' );
+}
```

Comment:
Thanks for the docs :).  I'm definitely going to put those in. Also TIL about ReferenceError!

> Another thing to think about that may or may not come into play here is that Object.assign() sets values while destructuring assignment defines them.

`window.config` is a function here.  I don't think spread works for those...does it?  I usually prefer spread/object literal when possible.